### PR TITLE
Fix your-badge link

### LINKED ISSUE #1954: Broken heading anchors
Links with heading anchors no longer seem to work on our homepage, for instance:
https://shields.io/#your-badge

As a consequence, our Readme link named _Make your own badges!_ is broken and shows a blank page rather than bringing the user to the appropriate section on _shields.io_.

### PR DESCRIPTION
**Pull request by paulmelnikow**
Fix #1954

https://shields-staging-pr-3058.herokuapp.com/#your-badge

### CODE REVIEW

File: frontend/components/usage.js [hunk 1]
```diff
@@ -170,7 +170,7 @@ export default class Usage extends React.PureComponent {
       <section>
         <H2 id="your-badge">Your Badge</H2>
 
-        <H3 id="static-badge">Static</H3>
+        <H3>Static</H3>
```
Comment (paulmelnikow) [inline frontend/components/usage.js]: As far as I know, this id is unused.

**Comment by paulmelnikow** [comment]
Thanks for testing this!

Would you like to give Edge a try with a longer timeout?

**Comment by PyvesB** [comment]
It doesn't seem to have helped much. 😕

**Comment by paulmelnikow** [comment]
Okay, I'll put it back then. Hopefully as you said, it won't be an issue for very long.

**Pull request merged**